docs/user/intelmq-api.rst

```diff
@@ -0,0 +1,112 @@
+.. 
+   SPDX-FileCopyrightText: 2020 Birger Schacht
+   SPDX-License-Identifier: AGPL-3.0-or-later
+
+###########
+Intelmq Api
+###########
+
+intelmq-api is a `hug <http://hug.rest>`_ based API for the `intelmq <https://github.com/certtools/intelmq/>`_ project.
+
+**********************************
+Installing and running intelmq-api
+**********************************
+
+``intelmq-api`` requires at least `Python 3.6 <https://www.python.org/downloads/release/python-360/>`_.
```

Comment:
That's the same requirement as for IntelMQ itself. Do we need to add it here explicitly again?

Reply:
No, you're right - I replaced it and mentioned the dependency on intelmq